src/widgets/AutoCompletion/ui/AutoCompletionScreen.tsx

```diff
@@ -0,0 +1,124 @@
+import { useCallback, useContext } from 'react';
+
+import { ProgressBar } from './ProgressBar';
+
+import { appletModel } from '~/entities/applet';
+import { useBanners } from '~/entities/banner/model';
+import { AutoCompletionModel } from '~/features/AutoCompletion';
+import { SurveyContext, SurveyLayout, SurveyManageButtons } from '~/features/PassSurvey';
+import { ROUTES, Theme } from '~/shared/constants';
+import Box from '~/shared/ui/Box';
+import Text from '~/shared/ui/Text';
+import { useCustomNavigation, useCustomTranslation, useOnceEffect } from '~/shared/utils';
+
+export const AutoCompletionScreen = () => {
+  const { t } = useCustomTranslation();
+
+  const navigator = useCustomNavigation();
+
+  const context = useContext(SurveyContext);
+
+  const { addWarningBanner } = useBanners();
+
+  const { entityCompleted } = appletModel.hooks.useGroupProgressStateManager();
+
+  const { removeAutoCompletion } = AutoCompletionModel.useAutoCompletionStateManager();
+
+  const { state, startEntityCompletion, activityName } = AutoCompletionModel.useAutoCompletion();
```

Comment:
Not sure what the `state` means. Please consider local renaming

Reply:
Done